samples/application-conf/README.md

```diff
@@ -267,6 +267,16 @@ zunit_CodeCoverageHost | Headless Code Coverage Collector host (if not specified
 zunit_CodeCoveragePort | Headless Code Coverage Collector port (if not specified IDz will be used for reporting) | true 
 zunit_CodeCoverageOptions | Headless Code Coverage Collector Options | true
 
+### zCEE3.Properties
```

Comment:
@M-DLB can you please align to the standard language prefix, which is the base name of the language script in lower case?